sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/component.py

```diff
@@ -529,17 +530,4 @@ def _resolve_local_code(self):
                         f.write(COMPONENT_CODE_PLACEHOLDER)
                     yield Code(base_path=self._base_path, path=code)
```

Comment:
maybe merge with the next yield?

Reply:
sure

Reply:
as we don't need to create temp folder for most of cases, I refactor this (looks like the original version now), may look cleaner